src/_pytest/mark/structures.py

```diff
@@ -367,7 +368,7 @@ def __repr__(self):
         return "<NodeKeywords for node {}>".format(self.node)
 
 
-@attr.s(cmp=False, hash=False)
+@attr.s(hash=False, **attrs_no_eq)  # type: ignore
```

Comment:
what's the `# type: ignore` here? can we link a typeshed issue such that we know what / how to clean this up later

Reply:
mypy is just not smart enough to deduce the meaning of `**attrs_no_eq` and I doubt it's gonna be a priority for them to fix…

Reply:
ah yeah I checked, it confuses mypy's overload detection